src/analytics/src/main/scala/osmesa/analytics/oneoffs/StreamingAOIMonitor.scala

```diff
@@ -247,30 +259,39 @@ object StreamingAOIMonitor
 
 object AOIMonitorUtils extends Logging {
 
-  case class NotificationData(notificationId: String, userId: String, name: String, email: String)
+  case class NotificationUserChangeSummary(notificationId: String,
+                                           user: String,
+                                           editCount: Long,
+                                           changesetCount: Int)
 
   // notificationId, userId, geom, name, email
   type Notification = (String, String, jts.Geometry, String, String)
```

Comment:
Make this a case class.  Improves readability, allows better interaction with Dataset/DataFrame APIs.